src/main-client.ts

```diff
@@ -206,8 +206,11 @@ export class MainClient extends BaseRestClient {
   constructor(
     restClientOptions: RestClientOptions = {},
     requestOptions: AxiosRequestConfig = {},
+    useTestnet?: boolean,
```

Comment:
Can you move this into RestClientOptions?

Reply:
I'm also wondering about the WS client. Anything where there's a boolean for testnet or the user has to choose `spottest` instead of `spot`, maybe it's simpler to have a `useTestnet?: boolean` in `WSClientConfigurableOptions`. 

The rest of the logic in the WS client could then automatically know whether to use testnet or not, and the user just says "spot" instead of needing to know "spottest" is an option. Simpler that way perhaps...less for anyone using the SDK to think about - thoughts? @SebastianBoehler

Reply:
Moved it into the RestClientOptions, havent touched ws client tho would be breaking changes there as it is pased as param in some public methods

Reply:
https://github.com/tiagosiebler/binance/blob/master/src/websocket-client.ts?plain=1#L1987 `subscribeUsdFuturesUserDataStream` would be breaking change if isTestnet would be removed since used in the constructor then. Or we use it in the WSOptions and keep it in such places

Reply:
ah man, you're completely right, and these methods are inconsistent with the other subscribe*UserDataStream methods (the others don't have that param at the start). Seems these are the only two methods that expose this param, unless I missed anything else.

I think I'd prefer to implement this in the WS options but keep it in these places for now, so avoiding the breaking change for now but not making it worse.

In a future breaking change / major release I'll want to consolidate these to the unified standard. Would you mind working through that as part of this PR?

Reply:
Yeah had to double check everything as well until I got through all. 

Lets keep this PR open then and I might work through all of this on the weekend. Can we merge the other one then in the meantime? https://github.com/tiagosiebler/binance/pull/419